backend/src/main/resources/application.yml

```
@@ -18,4 +18,12 @@ spring:
   secret:
-    key: ${JWT_SECRET_KEY}
\ No newline at end of file
+    key: ${JWT_SECRET_KEY}
+
+oauth:
+  kakao:
+    client-id: d5a0b288522bd3c7103156f48be8806c
+    redirect-uri: http://localhost:8080/api/v1/auth/kakao/callback
+  naver:
+    client-id: 7T8kCcHJ6XM0vlOyDjDE
+    redirect-uri: http://localhost:8080/api/v1/auth/naver/callback
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)